eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -1,3 +1,34 @@
+# |  2 | Linux ARM        |         |               |           |         |     N/A    |       |   N/A   |               |        |
+# |  3 | Linux (x64)      |    +    |               |     +     |    +    |      +     |   +   |         |               |        |
+# |  4 | Linux (x64)      |         |               |           |    +    |      +     |   +   |    +    |               |        |
+# |  5 | Windows x86      |    +    |               |     +     |         |      +     |       |         |               |        |
+# |  6 | MacOS (x64)      |    +    |               |           |    +    |      +     |       |    +    |               |        |
+# |  7 | MacOS (x64)      |         |               |     +     |    +    |      +     |       |    +    |               |        |
+# |  8 | Linux (x64) GCC5 |    +    |               |           |         |      +     |       |         |               |        |
+# |  9 | Linux (x64) GCC5 |         |               |     +     |         |      +     |       |    +    |               |        |
+# | 10 | Linux ARM        |    +    |       +       |           |         |     N/A    |       |   N/A   |               |        |
+# | 11 | Linux ARM        |         |       +       |           |    +    |     N/A    |       |   N/A   |               |        |
+# | 12 | Linux (x64)      |    +    |       +       |           |    +    |      +     |   +   |    +    |               |        |
+# | 13 | Linux (x64)      |         |       +       |           |    +    |      +     |   +   |         |       +       |   +    |
+# | 14 | Windows x64      |         |       +       |     +     |         |      +     |       |    +    |               |        |
+# | 15 | MacOS (x64)      |    +    |       +       |           |    +    |      +     |       |    +    |               |        |
+# | 16 | MacOS (x64)      |         |       +       |           |    +    |      +     |       |         |               |        |
+# | 17 | Linux (x64) GCC5 |    +    |       +       |           |         |      +     |       |    +    |               |        |
+# | 18 | Linux (x64) GCC5 |         |       +       |           |         |      +     |       |         |               |        |
+# | 19 | Windows x64      |    +    |               |           |         |            |       |         |               |        |
+# | 20 |         G     E     N     E     R     A     T     E           A     R     T     I     F     A     C     T     S          |
+# +----+------------------+---------+---------------+-----------+---------+------------+-------+---------+---------------+--------+
+#
+# ATTENTION: We should not enable code coverage for Release configurations.
+#      They produce low numbers, and are basically unactionable, plus the error message they produce in this case is confusing.
+#
+# N/A - There's no libcurl, cmocka, or paho-mqtt on LinuxARM, so no unit tests or samples can be built.
```

Comment:
nit:
```suggestion
# N/A - There's no libcurl, cmocka, or paho-mqtt on Linux ARM, so no unit tests or samples can be built.
```
Or were you trying to match the matrix name below rather than how its listed in the table above?